diff -urNw systemd-258.1.orig/debian/changelog systemd-258.1/debian/changelog
--- systemd-258.1.orig/debian/changelog	2025-11-04 01:35:17.000000000 -0800
+++ systemd-258.1/debian/changelog	2025-11-06 11:50:55.192685337 -0800
@@ -1,3 +1,9 @@
+systemd (258.1-2ubuntu1mangos1) unstable; urgency=medium
+
+  * Enable systemd-updated and updatectl.
+
+ -- Soren Hansen <soren.hansen@mastercard.com>  Thu, 06 Nov 2025 11:29:05 -0800
+
 systemd (258.1-2ubuntu1) resolute; urgency=medium
 
   * Merge with Debian unstable (LP: #2130132). Remaining changes:
diff -urNw systemd-258.1.orig/debian/control systemd-258.1/debian/control
--- systemd-258.1.orig/debian/control	2025-11-03 06:02:38.000000000 -0800
+++ systemd-258.1/debian/control	2025-11-06 11:54:17.089591773 -0800
@@ -552,9 +552,9 @@
 Recommends: ${dlopen:Recommends},
 Suggests: ${dlopen:Suggests},
 Breaks: systemd-boot (<< 257.4-2~),
-        systemd (<< 257.7-2~),
+        systemd (<< 258~),
 Replaces: systemd-boot (<< 257.4-2~),
-          systemd (<< 257.7-2~),
+          systemd (<< 258~),
 Description: simple UEFI boot manager - tools
  systemd-boot (short: sd-boot) is a simple UEFI boot manager. It provides a
  textual menu to select the entry to boot and an editor for the kernel command
diff -urNw systemd-258.1.orig/debian/rules systemd-258.1/debian/rules
--- systemd-258.1.orig/debian/rules	2025-11-04 01:26:51.000000000 -0800
+++ systemd-258.1/debian/rules	2025-11-06 11:50:55.193600945 -0800
@@ -149,7 +149,8 @@
 	-Dpcre2=enabled \
 	-Dukify=enabled \
 	-Dbootloader=$(if $(filter amd64 i386 arm64 armhf loong64 riscv64,$(DEB_HOST_ARCH)),enabled,disabled) \
-	-Dsysupdate=enabled
+	-Dsysupdate=enabled \
+	-Dsysupdated=enabled
 else
 CONFFLAGS += \
 	-Daudit=disabled \
@@ -175,7 +176,8 @@
 	-Dukify=disabled \
 	-Dbootloader=disabled \
 	-Dbpf-framework=disabled \
-	-Dsysupdate=disabled
+	-Dsysupdate=disabled \
+	-Dsysupdated=disabled
 endif
 
 # BPF/BTF are only enabled and usable on a few architectures
diff -urNw systemd-258.1.orig/debian/systemd-container.install systemd-258.1/debian/systemd-container.install
--- systemd-258.1.orig/debian/systemd-container.install	2025-11-03 06:02:38.000000000 -0800
+++ systemd-258.1/debian/systemd-container.install	2025-11-06 11:50:55.193688764 -0800
@@ -1,6 +1,6 @@
 #!/usr/bin/dh-exec
 usr/bin/importctl
-<pkg.systemd.upstream> usr/bin/updatectl
+usr/bin/updatectl
 usr/bin/systemd-vmspawn
 usr/lib/systemd/system/systemd-vmspawn@.service
 usr/bin/machinectl
@@ -34,7 +34,7 @@
 usr/lib/systemd/system/dbus-org.freedesktop.import1.service
 usr/lib/systemd/system/dbus-org.freedesktop.machine1.service
 usr/lib/systemd/system/dbus-org.freedesktop.portable1.service
-<pkg.systemd.upstream> usr/lib/systemd/system/dbus-org.freedesktop.sysupdate1.service
+usr/lib/systemd/system/dbus-org.freedesktop.sysupdate1.service
 usr/lib/systemd/user/machine.slice
 usr/lib/systemd/user/machines.target
 <pkg.systemd.upstream> usr/lib/systemd/user/dbus-org.freedesktop.import1.service
@@ -57,17 +57,17 @@
 usr/share/dbus-1/system.d/org.freedesktop.import1.conf
 usr/share/dbus-1/system.d/org.freedesktop.machine1.conf
 usr/share/dbus-1/system.d/org.freedesktop.portable1.conf
-<pkg.systemd.upstream> usr/share/dbus-1/system.d/org.freedesktop.sysupdate1.conf
+usr/share/dbus-1/system.d/org.freedesktop.sysupdate1.conf
 usr/share/dbus-1/system-services/org.freedesktop.import1.service
 usr/share/dbus-1/system-services/org.freedesktop.machine1.service
 usr/share/dbus-1/system-services/org.freedesktop.portable1.service
-<pkg.systemd.upstream> usr/share/dbus-1/system-services/org.freedesktop.sysupdate1.service
+usr/share/dbus-1/system-services/org.freedesktop.sysupdate1.service
 usr/share/polkit-1/actions/io.systemd.mount-file-system.policy
 usr/share/polkit-1/actions/io.systemd.namespace-resource.policy
 usr/share/polkit-1/actions/org.freedesktop.import1.policy
 usr/share/polkit-1/actions/org.freedesktop.machine1.policy
 usr/share/polkit-1/actions/org.freedesktop.portable1.policy
-<pkg.systemd.upstream> usr/share/polkit-1/actions/org.freedesktop.sysupdate1.policy
+usr/share/polkit-1/actions/org.freedesktop.sysupdate1.policy
 usr/share/zsh/vendor-completions/_systemd-nspawn
 usr/share/zsh/vendor-completions/_sd_machines
 usr/share/zsh/vendor-completions/_machinectl
diff -urNw systemd-258.1.orig/debian/systemd-container.manpages systemd-258.1/debian/systemd-container.manpages
--- systemd-258.1.orig/debian/systemd-container.manpages	2025-11-03 06:02:38.000000000 -0800
+++ systemd-258.1/debian/systemd-container.manpages	2025-11-06 11:50:55.196428755 -0800
@@ -1,11 +1,11 @@
 #!/usr/bin/dh-exec
 debian/tmp/usr/share/man/man1/importctl.1
-<pkg.systemd.upstream> debian/tmp/usr/share/man/man1/updatectl.1
+debian/tmp/usr/share/man/man1/updatectl.1
 debian/tmp/usr/share/man/man1/systemd-vmspawn.1
 debian/tmp/usr/share/man/man1/mount.ddi.1
 debian/tmp/usr/share/man/man1/systemd-nspawn.1
 debian/tmp/usr/share/man/man5/org.freedesktop.import1.5
-<pkg.systemd.upstream> debian/tmp/usr/share/man/man5/org.freedesktop.sysupdate1.5
+debian/tmp/usr/share/man/man5/org.freedesktop.sysupdate1.5
 debian/tmp/usr/share/man/man5/systemd.nspawn.5
 debian/tmp/usr/share/man/man5/sysupdate.d.5
 debian/tmp/usr/share/man/man5/sysupdate.features.5
@@ -28,5 +28,5 @@
 debian/tmp/usr/share/man/man8/systemd-sysupdate-reboot.timer.8
 debian/tmp/usr/share/man/man8/systemd-sysupdate.service.8
 debian/tmp/usr/share/man/man8/systemd-sysupdate.timer.8
-<pkg.systemd.upstream> debian/tmp/usr/share/man/man8/systemd-sysupdated.8
-<pkg.systemd.upstream> debian/tmp/usr/share/man/man8/systemd-sysupdated.service.8
+debian/tmp/usr/share/man/man8/systemd-sysupdated.8
+debian/tmp/usr/share/man/man8/systemd-sysupdated.service.8
